deploy/openshift-clusters/roles/kcli/kcli-redfish/tasks/verify_stonith.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Verify and enable stonith globally
+
+- name: Enable stonith globally (run on first node)
+  ansible.builtin.shell: |
+    oc debug node/{{ node_names[0] }} -- chroot /host bash -c "pcs property set stonith-enabled=true"
+  register: pcs_stonith_enable_result
+  delegate_to: localhost
+  environment:
+    KUBECONFIG: "{{ proxy_kubeconfig | default(omit) }}"
+    HTTP_PROXY: "{{ proxy_http_proxy | default(omit) }}"
+    HTTPS_PROXY: "{{ proxy_https_proxy | default(omit) }}"
+    NO_PROXY: "{{ proxy_no_proxy | default(omit) }}"
+
+- name: Verify stonith is enabled
+  ansible.builtin.shell: |
+    oc debug node/{{ node_names[0] }} -- chroot /host bash -c "pcs property show stonith-enabled"
+  register: pcs_stonith_verify_result
+  delegate_to: localhost
+  environment:
+    KUBECONFIG: "{{ proxy_kubeconfig | default(omit) }}"
+    HTTP_PROXY: "{{ proxy_http_proxy | default(omit) }}"
+    HTTPS_PROXY: "{{ proxy_https_proxy | default(omit) }}"
+    NO_PROXY: "{{ proxy_no_proxy | default(omit) }}"
+
+- name: Display stonith status
+  debug:
+    msg: |
+      Stonith enable result: {{ pcs_stonith_enable_result.stdout | default('No output') }}
+      Stonith verification: {{ pcs_stonith_verify_result.stdout | default('No output') }}
```

Comment:
It appears we don't actually fail if the result leaves us with stonith still disabled. Is this intentional?